### PR TITLE
docs: add tag `vim.fs.exists()` and document suggested replacement

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: runtime/lua/vim/fs.lua [hunk 1]
```diff
@@ -667,4 +667,14 @@ function M.rm(path, opts)
...
 
+--- Does not "exist" lmao, use vim.uv.fs_stat instead.
+---
+--- Example:
+---
+--- ```lua
+--- if vim.uv.fs_stat(file) then
+---   vim.print("file exists")
+--- end
+function M.exists(path) end
```
[the remark this answers is not in the document]
Comment (zeertzjq) [reply runtime/lua/vim/fs.lua:678]: And there is another problem: This function is included in completion for `vim.fs`.

File: runtime/lua/vim/fs.lua [hunk 2]
```diff
@@ -667,4 +667,16 @@ function M.rm(path, opts)
...
 
+--- Does not exist, use vim.uv.fs_stat instead.
+---
+--- Example:
+---
+--- ```lua
+--- if vim.uv.fs_stat(file) then
+---   vim.print("file exists")
+--- end
+function M.exists()
```
[the remark this answers is not in the document]
Comment (dundargoc) [reply runtime/lua/vim/fs.lua:678]: Any suggestions?

[the remark this answers is not in the document]
Comment (dundargoc) [reply runtime/lua/vim/fs.lua]: OK, fixed

File: runtime/lua/vim/fs.lua [hunk 3]
```diff
@@ -1,3 +1,15 @@
+--- @brief <pre>help
+--- vim.fs.exists({file})                                           *vim.fs.exists()*
+--- This function does not exist. Instead, use |uv.fs_stat()|.
```
[the remark this answers is not in the document]
Comment (dundargoc) [reply runtime/lua/vim/fs.lua]: Wait what is this suggestion, it looks weird

[the remark this answers is not in the document]
Comment (dundargoc) [reply runtime/lua/vim/fs.lua]: Do you not want the tag?

**Review by justinmk — APPROVED** [review]

**Pull request merged**